### PR TITLE
[3.2] Replace more uses of OS names with {EL}

### PR DESCRIPTION
**Pull request by ekohl**
(cherry picked from commit 434d771c89581a056faa6abc3fc3a67b3f9c4559)

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Maybe @Lennonka can review this one.

**Comment by ekohl** [comment]
3.2 is not used in Satellite so this is purely upstream to future make cherry picks easier.

File: guides/common/modules/proc_importing-kickstart-repositories.adoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ endif::[]
 ----
 
 ifeval::["{rhel-version}" == "8"]
-. Create directories for {RHEL}Linux 8 AppStream and BaseOS Kickstart repositories:
+. Create directories for {RHEL} 8 AppStream and BaseOS Kickstart repositories:
```
Comment (maximiliankolb) [inline guides/common/modules/proc_importing-kickstart-repositories.adoc:31]:
```suggestion
. Create directories for {EL} 8 AppStream and BaseOS Kickstart repositories:
```

**Comment by maximiliankolb** [comment]
> 3.2 is not used in Satellite so this is purely upstream to future make cherry picks easier.

just for the record, orcharhino does not use Foreman 3.2 either.

File: guides/common/modules/proc_importing-kickstart-repositories.adoc (hunk 1 above)
Comment (ekohl) [reply guides/common/modules/proc_importing-kickstart-repositories.adoc:31]: Below it is the instruction: `# mkdir --parents /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/kickstart` which also mentions rhel in the path name. I think `{RHEL}` is correct here.

**Review by maximiliankolb — APPROVED** [review]

**Pull request merged**